packages/basebridge/src/test-transactions.ts

```diff
@@ -0,0 +1,374 @@
+    'bridge erc20 from ethereum to base, when tokenAddress and contractAddress not defined',
+    { tokenAddress: undefined, contractAddress: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_ETHEREUM_TO_BASE_TEST,
+    'bridge erc20 from ethereum to base, when recipient not defined',
+    { recipient: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_ETHEREUM_TO_BASE_TEST,
+    'bridge erc20 from ethereum to base, when amount not defined',
+    { amount: undefined },
+  ),
+  // eth from base
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum',
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when contractAddress not defined',
+    { contractAddress: undefined },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when tokenAddress not defined',
+    { tokenAddress: undefined },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when tokenAddress and contractAddress not defined',
+    { tokenAddress: undefined, contractAddress: undefined },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when recipient not defined',
+    { recipient: undefined },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when amount not defined',
+    { amount: undefined },
+  ),
+  // erc20 from base
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'when bridge erc20 from base to ethereum',
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when contractAddress not defined',
+    { contractAddress: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when tokenAddress not defined',
+    { tokenAddress: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when tokenAddress and contractAddress not defined',
+    { tokenAddress: undefined, contractAddress: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when recipient not defined',
+    { recipient: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when amount not defined',
+    { amount: undefined },
+  ),
+]
+
+export const failingTestCases = [
+  // eth from ethereum
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when sourceChainId is not correct',
+    {
+      sourceChainId: 5,
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when amount is not correct',
+    {
+      amount: '4000000',
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
```

Comment:
Would remove this test. Since this will always match the ABI for an ETH deposit and not consider the token address.